cdap-ui/app/cdap/components/Experiments/FieldLevelLineage/Table/Table.css

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+*/
+
+.table {
+  width: 170px;
+  border: 1px solid #bbbbbb;
+  margin: 10px;
+}
+
+.row {
```

Comment:
We should try to avoid restyling bootstrap classes. This is generally an anti-pattern. When we rewrite this in css-in-js I believe we will get rid of this style or restructure this to something else?